### PR TITLE
[engine] Add ability to enable global rows meta filtering

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: docs/modules/configuration/pages/tests-configuration.adoc [hunk 1]
```diff
@@ -142,6 +142,10 @@ e.g., `0`, `#`, specify the way indices are displayed.
 |`false`
 |Skips the story in case of failure in the given story
 
+|`bdd.configuration.meta-by-row`
```
Comment (valfirst) [inline docs/modules/configuration/pages/tests-configuration.adoc:145]:
```suggestion
|`engine.meta-by-row-enabled`
```

File: docs/modules/configuration/pages/tests-configuration.adoc [hunk 2]
```diff
@@ -142,6 +142,10 @@ e.g., `0`, `#`, specify the way indices are displayed.
...
+|`false`
+|Enabled filtering of scenario by meta defined in examples table rows
```
Comment (valfirst) [inline docs/modules/configuration/pages/tests-configuration.adoc:147]: it's not really clear, actually it doesn't enable filtering, it allows to set different meta per ExamplesTable row

File: vividus-tests/src/main/resources/story/integration/Use rows meta filtering.story [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Meta:
+    @epic vividus-engine
+
+Scenario: Should filter rows by meta
+Then `<miestas>` is equal to `Vankuveris`
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/Use rows meta filtering.story:5]: Vancouver

File: vividus-tests/src/main/resources/story/integration/Use rows meta filtering.story [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+Examples:
+|Meta:     |miestas   |
+|@locale ca|Vankuveris|
+|@locale lt|Vilnius   |
+|@locale in|Mumbajus  |
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/Use rows meta filtering.story:10]: Mumbai

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/configuration/pages/tests-configuration.adoc (hunk 2 above)
Comment (uarlouski) [reply docs/modules/configuration/pages/tests-configuration.adoc:147]: No, you can set meta even without this prop but it won't be taken into consideration when filtering stories
image

File: docs/modules/configuration/pages/tests-configuration.adoc [hunk 5]
```diff
@@ -142,6 +142,10 @@ e.g., `0`, `#`, specify the way indices are displayed.
 |`false`
 |Skips the story in case of failure in the given story
 
+|`engine.meta-by-row-enabled`
```
Comment (valfirst) [inline docs/modules/configuration/pages/tests-configuration.adoc:145]:
```suggestion
|`engine.filtering-by-table-meta-enabled`
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**